wordle-web/components/settings-dialog.vue

```diff
@@ -1,9 +1,91 @@
 <template>
   <div>
+
+     <v-btn icon @click="helpDialog">
+      <v-icon> mdi-help </v-icon>
+    </v-btn>
+
+    <v-btn icon @click="aboutDialog">
+      <v-icon> mdi-hamburger</v-icon>
+    </v-btn>
+
     <v-btn icon @click="toggleDialog">
       <v-icon> mdi-cog </v-icon>
     </v-btn>
 
+   
+
+     <v-dialog v-model="about" width="450">
+      <v-card>
+        <v-container>
+          <v-card-title> Menu </v-card-title>
+
+          <v-card-text>
+            <v-menu offset-y>
+              <template #activator="{ on, attrs }">
+                <v-btn color="primary" dark v-bind="attrs" v-on="on" nuxt to="/about">
+                  About
+                </v-btn>
+              </template>
+            </v-menu>
```

Comment:
v-menu is not typically used in this manner. Take a look at this PR comment, as it's a similar problem!
https://github.com/IntelliTect-Samples/EWU-CSCD379-2022-Spring/pull/38/files#r865522913